opensfm/commands/detect_features.py

```diff
@@ -68,10 +78,14 @@ def detect(args):
     logger.info('Extracting {} features for image {}'.format(
         data.feature_type().upper(), image))
 
-    start = timer()
+    config = deepcopy(data.config)
```

Comment:
Can we avoid doing the copy? Maybe passing the `size` and `min_frames` as a parameter to extract features. Or passing an `is_panorama` boolean.